Assets/HoloToolkit/Utilities/Editor/ConfigureMenu.cs

```diff
@@ -191,52 +191,62 @@ private static void EnableVirtualReality()
             }
         }
 
-        [MenuItem("HoloToolkit/Configure/HoloLens Recommended Capabilities ", priority = 1)]
-        static void HoloLensRecommendedCapabilities()
+        [MenuItem("HoloToolkit/Configure/Apply HoloLens Capability Settings", priority = 1)]
+        static void ApplyHoloLensCapabilitySettings()
         {
-            AutoConfigureMenu window = (AutoConfigureMenu)EditorWindow.GetWindow(typeof(AutoConfigureMenu), true, "HoloLens Recommended Capabilities");
+            AutoConfigureMenu window = (AutoConfigureMenu)EditorWindow.GetWindow(typeof(AutoConfigureMenu), true, "Apply HoloLens Capability Settings");
             window.Show();
         }
 
         void OnGUI()
         {
-            capability(PlayerSettings.WSACapability.Microphone, @"  Microphone
-Required for access to the HoloLens microphone. 
+            capabilityToggle(PlayerSettings.WSACapability.Microphone, @"Microphone
+
+Required for access to the HoloLens microphone.
```

Comment:
Nit: @ strings are difficult to maintain, better off just throwing this into a .resx, which makes localization later much more palatable.